xep-0363.xml

```diff
@@ -210,6 +221,7 @@
   </error>
 </iq>]]></example>
 <p>For any other type of error the service SHOULD respond with appropriate error types to indicate temporary or permanent errors.</p>
+<p>For temporary errors such as exceeding a personal quota the service MAY include a &lt;retry/&gt; element qualified by the urn:xmpp:http:upload:0 namespace as a child of the &lt;error/&gt; element. That element MUST include an attribute 'stamp' which indicates the time at which the requesting entity MAY try again. The format of the timestamp must adhere to the date-time format specified in &xep0082; and MUST be expressed in UTC. The service SHOULD NOT impose sanctions on an entity simply for retrying earlier than the specified time.</p>
```

Comment:
* s/That element MUST/The retry element MUST/
* s/requesting entity MAY try again/requesting entity may try again/ (this is passively describing what the entity may be doing, so I don't think it's actually 2119 language?)
* "entity ~simply~ for retrying earlier"